portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,45 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+#header {
+    font-family:monospace;
+    font-size:300%;
+    margin-top:20%;
+    margin-left:30%;
```

Comment:
this applies to all styles in this file:
1) the indentation level should be consistent (2 spaces, like the li class)
2) you should alphabetize the styles within the class (i.e. background-color before display)